src/View3d.ts

```diff
@@ -793,7 +793,11 @@ export class View3d {
    * @param {boolean} enabled
    */
   setVolumeChannelEnabled(volume: Volume, channel: number, enabled: boolean): void {
-    this.image?.setVolumeChannelEnabled(channel, enabled);
+    // NOTE: explicitly calling `image?.setVolumeChannelEnabled` allowed the enabled/disabled state of the channel to
+    //   be inconsistent across loads (incl. on time or z-slice changes) because channel state is restored on reload
+    //   from the object managed by `setChannelOptions`.
```

Comment:
Does this note still make sense to leave here now that the offending line is removed? Does it make sense to leave it in but commented-out?

Reply:
I wrote this more as an explanatory comment on the TODO below, so that there's a history that motivates why we ought to examine how state is managed within channels. So I suppose it might be more at home in a GitHub issue?